util/test/sub_test

```diff
@@ -298,7 +296,10 @@ def cleanup(execname):
                 cmd = [lsof, execname]
                 sys.stdout.write('[Inspecting open file handles with: {0}\n'.format(' '.join(cmd)))
                 subprocess.Popen(cmd).communicate()
-    return
+
+        # Do not print the warning for cygwin32 when errno is 16 (Device or resource busy).
```

Comment:
I might add some more (either to this comment or in the pr) about why we're doing this.